### PR TITLE
Use paths that are more "environment-friendly"

### PR DESCRIPTION
**Pull request by andsild**
Regarding commit 39a89f4: 
http://stackoverflow.com/a/16365367
http://stackoverflow.com/a/21613044

Commit 0282b16:
/usr/lib doesn't exist on my system so the test fails (NixOS). /usr/bin has to exist on all UNIX systems

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (Shougo) [inline run_tests.sh]: Why did you change it?
I don' think this change is needed.

Comment (tweekmonster) [inline run_tests.sh]: I've run into cases where `/bin/bash` doesn't actually exist and `/usr/bin/env bash` was more reliable.  I think it was on FreeBSD.

OTH, the contents of `run_tests.sh` could be placed in `Makefile`.

**Comment by Shougo** [comment]
Thank you.  I have fixed.
